### ISSUE TITLE
Alert: role-alert announcement should be deactivated in some cases

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5222]
In some cases in portal we use visual warnings or visual info box, they are currently read as priority text by screen readers because they have always `role="alert"`.

In some cases, it is necessary for DXP functionality that we do not provide `role="alert"`, example:
https://www.figma.com/file/nVGg94HV43eQVkjsEl6bft/LPS-167668---New-pattern-for-fragments-mapped-to-entities-the-user-does-not-have-permissions-on?node-id=26%3A131059&t=Dv8ZlHcL2WZTelQx-0

<img width="1074" alt="image" src="https://user-images.githubusercontent.com/7963804/204851098-49eec478-099b-4dbc-9166-8e2444ae2cb9.png">

My proposal is:
- Add a new API property for Alert compoent 
- Make `role="alert"` the default value
- Provide by API a way to desactivate it, for example: `roleAlert="false"`

cc @liferay-lexicon
